test/unit/test_status.py

```diff
@@ -0,0 +1,34 @@
+import mocks
+from fake_statuses import MINIMAL_JSON, MINIMAL_STATUS, SNAPPASS_JSON
```

Comment:
`fake_statuses` is more obviously something local than `mocks` but I suggest the same thing here: `from .fake_statuses import ...`.